tests/production/messages/get_message/test_success.py

```diff
@@ -0,0 +1,21 @@
+import os
+import requests
+import pytest
+from lib import Assertions, Authentication
+from lib.constants.constants import PROD_URL
+from lib.constants.messages_paths import MESSAGES_ENDPOINT, SUCCESSFUL_MESSAGE_IDS
+
+
+@pytest.mark.devtest
```

Comment:
`prodtest` - plus the `nhsd_apim_authorization` line needs to be removed or this will error when run on the release pipeline